core/src/main/java/nl/nn/adapterframework/processors/CorePipeLineProcessor.java

```diff
@@ -52,29 +50,26 @@
 /**
  * @author Jaco de Groot
  */
-public class CorePipeLineProcessor implements PipeLineProcessor, ApplicationContextAware {
+public class CorePipeLineProcessor implements PipeLineProcessor {
 	private Logger log = LogUtil.getLogger(this);
 	private @Setter PipeProcessor pipeProcessor;
-	private @Setter ApplicationContext applicationContext;
+	private @Setter AdapterManager adapterManager;
```

Comment:
En hoe wordt deze nou gezet? Ik zie dat hij in SpringApplicationContext gedefinieerd is. Wordt elke bean die in zo'n file gezet is automatisch geautowired?